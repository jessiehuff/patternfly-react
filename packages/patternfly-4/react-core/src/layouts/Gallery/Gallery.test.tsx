import * as React from 'react';
import { Gallery } from './Gallery';
import { shallow } from 'enzyme';

[true, false].forEach(hasGutter => {
  test(`gutter ${hasGutter}`, () => {
    const view = shallow(<Gallery hasGutter={hasGutter} />);
    expect(view).toMatchSnapshot();
  });
});
