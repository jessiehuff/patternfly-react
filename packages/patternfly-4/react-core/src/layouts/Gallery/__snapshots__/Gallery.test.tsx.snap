// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gutter false 1`] = `
<div
  className="pf-l-gallery"
/>
`;

exports[`gutter true 1`] = `
<div
  className="pf-l-gallery pf-m-gutter"
/>
`;
